packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -48,7 +48,36 @@ export function UserHeaderItem(props: PropsWithChildren<UserHeaderItemProps>) {
     const [showAboutModal, setShowAboutModal] = useState(false);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
-    const AccountIcon = user.impersonated ? <Account color="info" /> : <Account />;
+    const AccountIcon = user.impersonated ? (
+        <Box display="flex">
+            <Avatar
+                sx={{
```

Comment:
Here is the same issue like above, it would be nicer if you inheritance, the component from Mui